api/src/test/java/org/openmrs/validator/AlertValidatorTest.java

```diff
@@ -37,7 +37,7 @@ public void setUp() {
 		
 		errors = new BindException(alert, "alert");
 	}
-	
+
 	@Test
 	public void shouldFailValidationIfAlertTextIsNull() {
```

Comment:
@Wandji69 what are you trying to test here?
there is no method body

Reply:
I’ll look into this but I don’t remember writing any test here.

Reply:
@Wandji69 Kindly do git stash to revert the changes. That does happen sometimes!